src/kex/kex.c

```diff
@@ -6,6 +6,7 @@
 #include <oqs/kex_rlwe_msrln16.h>
 #include <oqs/kex_lwe_frodo.h>
 #include <oqs/kex_sidh_cln16.h>
+#include <oqs/kex_code_mcbits.h>
```

Comment:
This should be wrapped with a #ifdef ENABLE_CODE_MCBITS. The Visual Studio project doesn't currently copy "kex_code_mcbits.h" into the "oqs" dir; this line therefore breaks the Windows build.